### PR TITLE
Add site to monorepo

### LINKED ISSUE #547: Dev: Setup docusaurus in monorepo
Add an out-of-the-box [Docusaurus](https://docusaurus.io/) (as per decision in #548) setup to the `uitk` monorepo.

By the end of this developers should be able to:

* Run the site locally on their machines for development work
* Create a static build of the site that can be deployed somewhere

Note: As part of this task we need to figure out if/how the Docusaurus builds can be integrated with the `modular` build tooling we use for everything else.

### PR DESCRIPTION
**Pull request by EleanorHaproff**
Add a new Docusaurus site, along with start and build commands. 

Closes #547

### CODE REVIEW

**Comment by james-nash** [comment]
The preview links in the auto-generated Cloudflare comment above seem to point to Storybook. Just want to confirm that's expected at this point (since we have #550 to setup hosting later on), right?

**Review by james-nash — COMMENTED** [review]
The changes look fine. Just wondering if we want to strip out the Docusaurus example pages and images at this point, or do that as part of subsequent issues?

**Comment by EleanorHaproff** [comment]
> The preview links in the auto-generated Cloudflare comment above seem to point to Storybook. Just want to confirm that's expected at this point (since we have #550 to setup hosting later on), right?

That is correct :)

**Comment by EleanorHaproff** [comment]
> The changes look fine. Just wondering if we want to strip out the Docusaurus example pages and images at this point, or do that as part of subsequent issues?

Yes I was wondering that... I was planning on doing it for subsequent issues yeah but happy to do it as part of this one if you think that's better. Shouldn't be a big job

**Comment by james-nash** [comment]
> Yes I was wondering that... I was planning on doing it for subsequent issues yeah but happy to do it as part of this one if you think that's better. Shouldn't be a big job

It may be worth doing it now. If we can strip it down to just have our planned 4 pages (Home, Getting started, Components & Contributing), then I think that might help the rest of the team better understand the skeleton that we now have to build upon. Those pages could each be empty or just have a paragraph of Lorem Ipsum. Otherwise, if we keep the out-of-the-box sample content, we might have a lot of handwavy explaining to do "imagine this bit gone, and our page would be here, etc. etc.".

That being said, I don't have particularly strong feelings about this, so happy to defer if you think makes more sense.

**Comment by ZarkoPernar** [comment]
I agree with @james-nash. Stripping out the dummy pages will make this PR easier to review.

**Comment by james-nash** [comment]
> [🔥 blog](https://github.com/jpmorganchase/uitk/pull/595/commits/bc2072c8df0def9e9ae400b5934bce770a90e687)

Love the commit message!

File: site/docusaurus.config.js
Comment (joshwooding) [inline site/docusaurus.config.js:10]:
```suggestion
  tagline: "JP Morgan - UI Toolkit",
```

File: site/package.json
Comment (origami-z) [inline site/package.json:23]: May be good to keep same with root

**Review by james-nash — APPROVED** [review]
If you could chuck in another "Contributing" placeholder page before merging, that would be great.

But, approving now as it looks fine otherwise. Nice work!

**Review by JoanaMMoreira — APPROVED** [review]

File: site/package.json
Comment (EleanorHaproff) [reply site/package.json:23]: Have changed this on @JoanaMMoreira's branch [website-initial-styling](https://github.com/jpmorganchase/uitk/compare/website-initial-styling?expand=1)

**Review by JoanaMMoreira — APPROVED** [review]

**Review by james-nash — APPROVED** [review]
LGTM 👍

**Pull request merged**
